internal/publisher/publisher.go

```diff
@@ -0,0 +1,96 @@
+package publisher
+
+import (
+	"encoding/json"
+	"fmt"
+	"log/slog"
+	"time"
+
+	"github.com/EvergenEnergy/remote-standby/internal/config"
+	"github.com/EvergenEnergy/remote-standby/internal/plan"
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+)
+
+type Service struct {
+	logger     *slog.Logger
+	cfg        config.Config
+	mqttClient mqtt.Client
+}
+
+func NewService(logger *slog.Logger, cfg config.Config, mqttClient mqtt.Client) *Service {
+	return &Service{
+		logger:     logger,
+		cfg:        cfg,
+		mqttClient: mqttClient,
+	}
+}
+
+type CommandPayload struct {
+	Action string  `json:"action"`
+	Value  float64 `json:"value"`
+}
+
+type ErrorPayload struct {
+	Category  string    `json:"category"`
+	Message   string    `json:"message"`
+	Timestamp time.Time `json:"timestamp"`
+}
+
+const errorCategory = "Standby"
```

Comment:
I believe it should be `errorCategoryStandby` - to make it more obvious how to extend these values in the future.